azure-pipelines/build/templates/dopapt-docker-steps.yml

```diff
@@ -2,19 +2,52 @@
 # Consume this steps template in one or more jobs by passing in parameter values.
 
 parameters:
-- name: targetOsArch    # example: debian10-arm32
+- name: targetOS        # example: ubuntu1804
   type: string
-- name: dockerImageName # example: jimsonmsft/debian10-arm32:latest
+- name: targetOsArch    # example: ubuntu1804-x64
   type: string
-- name: config          # debug/release
+- name: config          # debug/release/minsizerel/relwithdebuginfo
   type: string
 
 steps:
 - checkout: self
 
+# TODO(jimson): Once these are onboarded as an artifact to Image Factory, can remove all dependency install steps
 - task: CmdLine@2
   inputs:
-    script: 'sudo docker run --rm --entrypoint=/bin/bash -v $(Build.SourcesDirectory):/code -v /tmp/build-deliveryoptimization-plugin-apt-${{parameters.targetOsArch}}:/tmp/build-deliveryoptimization-plugin-apt ${{parameters.dockerImageName}} "/code/build/docker/docker-build-plugin.sh" "/code" "${{parameters.config}}"'
+    script: 'chmod +x bootstrap.sh'
+    workingDirectory: '$(Build.SourcesDirectory)/build'
+  displayName: 'chmod bootstrap script'
+
+- task: CmdLine@2
+  inputs:
+    script: 'sudo ./bootstrap.sh --platform ${{parameters.targetOS}} --install containertools'
+    workingDirectory: '$(Build.SourcesDirectory)/build'
+  displayName: 'Install Docker'
```

Comment:
>displayName: 'Install Docker' [](http://example.com/codeflow?start=2&length=29)

Why does only the plugin arm build require docker to be installed?

Reply:
Oops, forgot to remove